pkg/cmd/api/api.go

```diff
@@ -240,19 +243,36 @@ func magicFieldValue(v string, stdin io.ReadCloser) (interface{}, error) {
 }
 
 func readUserFile(fn string, stdin io.ReadCloser) ([]byte, error) {
-	r, err := openUserFile(fn, stdin)
-	if err != nil {
-		return nil, err
+	var r io.ReadCloser
+	if fn == "-" {
+		r = stdin
+	} else {
+		var err error
+		r, err = os.Open(fn)
```

Comment:
Does the err need to be declared above?

```suggestion
		r, err := os.Open(fn)
```

Reply:
Yes, because `r, err :=` would also initialize a new variable called `r` that would shadow the one declared in the outer scope, but I wanted to set the outer scope one.